contracts/mainnet/connectors/morpho-compound/main.sol

```diff
@@ -0,0 +1,416 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.7.0;
+pragma experimental ABIEncoderV2;
+import "./helpers.sol";
+import "./events.sol";
+
+abstract contract MorphoCompound is Helpers, Events {
+	/**
+	 * @dev Deposit ETH/ERC20_Token.
+	 * @notice Deposit a token to Morpho Compound for lending / collaterization.
+	 * @param _tokenAddress The address of underlying token to deposit.(For ETH: 0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE)
+	 * @param _poolTokenAddress The address of cToken to deposit.(For ETH: cWETH address)
```

Comment:
@shriyatyagii It's cETH address? Instead of cWETH address? Or Morpho has cWETH?

Reply:
No, it's cETH, my bad

Reply:
fixed it